api/api.yaml

```diff
@@ -191,6 +191,26 @@ paths:
           $ref: '../result/request.yaml#/components/responses/BadRequest'
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
+  
+  /control/protocols/lacp:
```

Comment:
This approach has the potential of significantly increasing the size of the api surface.  A single API with the ability to extend the model should be proposed.

Reply:
Agree. Moved under a generic API